source/docs/essentials/add-a-page-client-side.md

```diff
@@ -1,15 +1,11 @@
 ---
 id: add-a-page-client-side
 title: Add a new page
+description:
+  Front-Commerce comes with a classic set of pages for an e-commerce application. It has URLs for Catalog routes (Categories, Product, Search…), Checkout routes, Account routes, Cms pages…
+  When building your own e-commerce experience you will most likely need to add your own pages/routes. That's what we will focus on in this guide.
```

Comment:
The same comment for linebreaks, this is fine if they are supported, otherwise online them.

Reply:
Done. Thanks.